packages/flutter/lib/src/widgets/basic.dart

```diff
@@ -5961,6 +5963,13 @@ class RawImage extends LeafRenderObjectWidget {
   /// If non-null, this color is blended with each image pixel using [colorBlendMode].
   final Color? color;
 
+  /// If non-null, the value from the [Animation] is multiplied with the opacity
+  /// of each image pixel before painting onto the canvas.
+  ///
+  /// This is more efficient that using [FadeTransition] to change the opacity
+  /// of an image.
```

Comment:
Users might appreciate hearing about why - it avoids creating a new layer/render target.

This should be fine too because it's a leaf render object, so we don't have to worry about compositing it with children that might also need alpha blending.